src/Stringy.php

```diff
@@ -1250,6 +1250,18 @@ public function split($pattern, $limit = null)
     }
 
     /**
+     * Strip all whitespace characters. This includes tabs and newline
+     * characters, as well as multibyte whitespace such as the thin space
+     * and ideographic space.
+     *
+     * @return Stringy Object with whitespace stripped
+     */
+    public function stripWhitespace()
+    {
+        return $this->regexReplace('[[:space:]]+', '');
```

Comment:
Thanks for the PR! I'm a bit on the fence with whether or not it's necessary. Is it that common, and that much more convenient over writing `$str->regexReplace('[[:space:]]+', '');`